### PR TITLE
OI-297 FAQ

### PR DESCRIPTION
**Pull request by nvelychenko**

### CODE REVIEW

File: modules/openideal_faq/openideal_faq.install [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+
+  $faqs = $node_storage->loadByProperties(['type' => 'faq']);
+  $node_storage->delete($faqs);
+
+  // Delete FAQ Content type if exists.
+  if ($faq = \Drupal::entityTypeManager()->getStorage('node_type')->load('faq')) {
+    $faq->delete();
+  }
+
```
Comment (db-Matroskeen) [inline modules/openideal_faq/openideal_faq.install:33]: There is an extra empty row.

File: modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file Contains main hooks for ofc module.
```
Comment (db-Matroskeen) [inline modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module]: 1) Let's put the full module name;
2) Description should starts from new line (below @file)

File: modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ *
+ * Dirty save path alias that is hardcoded to the node uuid that is
+ * imported with this module, because of update_helper cannot save
+ * path alias.
+ */
+function openideal_faq_content_node_insert(NodeInterface $node) {
+  if ($node->uuid() == '62e92783-c1a6-4cc2-bd50-198884f083ec') {
+    \Drupal::service('path.alias_storage')->save("/node/" . $node->id(), "/faq", "en");
```
Comment (db-Matroskeen) [inline modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module]: Let's use single quotes to be consistent.

**Review by db-Matroskeen — CHANGES_REQUESTED** [review]
Thanks for making a new branch - this is much easier to review.
I have just a few minor nits. Looks good overall.

File: modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module (hunk 2 above)
Comment (nvelychenko) [reply modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module]: Done.

File: modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module (hunk 3 above)
Comment (nvelychenko) [reply modules/openideal_faq/modules/openideal_faq_content/openideal_faq_content.module]: Done.

**Pull request merged**